src/Models/ProjectLabels.coffee

```diff
@@ -0,0 +1,31 @@
+BaseModel = require '../BaseModel'
+Utils = require '../Utils'
+
+class ProjectLabels extends BaseModel
+  list: (projectId, params = {}, fn = null) =>
+    @debug "ProjectLabels::labels()"
+
+    if 'function' is typeof params
+      fn = params
+      params = {}
+    params.page ?= 1
+    params.per_page ?= 100
+
+    do (->
```

Comment:
Check out the way I've done this in Projects.coffee:

https://github.com/node-gitlab/node-gitlab/blob/develop/src/Models/Projects.coffee#L24

I originally wrote it the way you've done here (I assume you copy/pasted), but figured out a cleaner way a month or so back.